### DISCUSSION TITLE
I cant create a private projet

### DISCUSSION THREAD

**Discussion by majdroses21** [community/community#118343] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

When I was in Syria, it was forbidden to start a private project. Why can't I start a private project now?

**Comment by alizardguy** [comment]
It seems to be a legal regulation according to [the docs](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls) and a [post I found](https://github.com/orgs/community/discussions/58614).

**Comment by DeathKnight813** [comment]
It seems that your account was marked as spam or you set wrong settings.

**Comment by Sackitey68** [comment]
Check your account very well for spam
